FG-100EF.yaml

```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100EF
+part_number: FG-100EF
+slug: fg-100ef
+is_full_depth: false
+u_height: 1

```

Comment:
Please add `airflow` if you have/can find it

Reply:
Airflow added.